tools/android/MASTG-TOOL-0103.md

```diff
@@ -0,0 +1,19 @@
+---
+title: uber-apk-signer
+platform: android
+source: https://github.com/patrickfav/uber-apk-signer
+---
+
+A tool that helps to sign, [zip aligning](https://developer.android.com/studio/command-line/zipalign.html) and verifying multiple Android application packages (APKs) with either debug or provided release certificates (or multiple). It supports v1, v2, [v3](https://source.android.com/security/apksigning/v3) and [v4](https://source.android.com/security/apksigning/v4) Android signing schemes. Easy and convenient debug signing with embedded debug keystore. Automatically verifies signature and zipalign after every signing.
+
+Download the jar from the [latest release](https://github.com/patrickfav/uber-apk-signer/releases/latest) and run:
+
+```bash
+java -jar uber-apk-signer.jar --apks /path/to/apks
+```
+
+Demo:
+
+[![asciicast](https://asciinema.org/a/91092.png)](https://asciinema.org/a/91092)
```

Comment:
Do we want to go this route, vs including an example ourselves?

Reply:
This is fine, whenever there are examples available we can include them. If there are videos, even better!